src/containers/group-management/group-detail.tsx

```diff
@@ -130,6 +136,10 @@ class GroupDetail extends React.Component<RouteComponentProps, IState> {
   }
 
   render() {
+    if (this.state.redirect) {
+      return <Redirect to={this.state.redirect} />;
```

Comment:
:+1:, but also https://github.com/ansible/ansible-hub-ui/pull/859#issuecomment-910920191

```suggestion
      return <Redirect push to={this.state.redirect} />;
```

Reply:
Ok.